examples/simple-nft-minter/index.mjs

```diff
@@ -0,0 +1,70 @@
+import { Reach, test } from '@reach-sh/stdlib';
+const stdlib = new Reach(process.env, { REACH_NO_WARN: 'Y' });
+
+const startingBalance = stdlib.parseCurrency(100);
+
+const [minter, receiver] = await stdlib.newTestAccounts(2, startingBalance);
+
+const gasLimit = 500000;
+if (stdlib.connector != 'ALGO') {minter.setGasLimit(gasLimit)};
+if (stdlib.connector != 'ALGO') {receiver.setGasLimit(gasLimit)};
+
+const mintAddr = minter.getAddress();
+console.log(`Your address is ${mintAddr}`);
+const recAddr = receiver.getAddress();
+console.log(`receiver's address is ${recAddr}`);
+
+const minterAddrFormat = await stdlib.formatAddress(minter);
+console.log(`The minter's formatted address is ${minterAddrFormat}`);
+const receiverAddrFormat = await stdlib.formatAddress(receiver);
+console.log(`The receiver's formatted address is ${receiverAddrFormat}`);
+
+const fmt = (x) => stdlib.formatCurrency(x, 4);
+const getBal = async (who) => fmt(await stdlib.balanceOf(who));
+const bal = await getBal(minter);
+console.log(`Minter starting balance: ${bal}.`);
+
+const name = "JPAlgos";
+const symbol = "JPA";
+
+const opts = { 
+    supply: 1, 
+    au: "ipfs://bafybeigdyrzt5...", //asset url
```

Comment:
This should be `url`. `au` is algorand specific and we don't acknowledge this field when launching the token.

Reply:
How would I learn this information?

Reply:
That we don't support `au`? It's not listed under https://docs.reach.sh/frontend/#js_stdlib.launchToken & we only support what we say in the docs 😉